src/writeback/changes.rs

```diff
@@ -1,7 +1,11 @@
 use super::WriteBackError;
-use super::properties::PropertyFile;
+use super::properties::{ColumnProperty, ModelProperty, PropertyFile};
 use crate::change_descriptors::ModelChange;
-use std::path::Path;
+use crate::config::ModelPropertiesLayout;
+use dbt_serde_yaml;
```

Comment:
red flag, don't think I should use the serde unless in rust writeback.  Also, this is a useless import.